index.js

```diff
@@ -98,9 +98,10 @@ module.exports = function makeIPFSFetch ({ ipfs }) {
           path,
           content: body
         }, {
+          cidVersion: 1,
           wrapWithDirectory: true
         })
-        const cidHash = cidToString(cid)
```

Comment:
iirc encodings are specific to namespaces:

- base32 is default for CIDs in `/ipfs/` – cidv1 with sha-256 represented in base32 will fit in DNS label just fine :)
- base36 is default for IPNS keys in `/ipns/` – you only need to use base36 with CIDv1 representing libp2p-key that is ED25519, because it is bit longer than sha-256 and base32 is too long, but base36 is just right.

We could use Base36 everywhere, but its bit less efficient than Base32, so we only use it where necessary (`/ipns/`)
This has a nie side effect of `/ipfs/` being `bafy...` and  `/ipns/` starting with distinct `k...`.

Reply:
That's a good point, thank you for going over it!